File: src/geek time/python/week01/70. Climbing Stairs.py
def climbStairs(n: int) -> int :
    if n < 3:
        return n
    f1 = 1
    f2 = 2
    for i in range(3, n + 1):
        f2 += f1
        f1 = f2 - f1
    return f2

if __name__ == "__main__":
    print(climbStairs(10))

File: src/geek time/typescript/week01/70. Climbing Stairs.ts
function climbStairs(n: number): number {
    if (n < 3) {
        return n;
    }

    let f1 = 1;
    let f2 = 2;
    for (let i = 3; i <= n; i++) {
        f2 += f1;
        f1 = f2 - f1;
    }
    return f2;
}

console.log(climbStairs(10));